appdata-fixes.patch

```diff
@@ -1,81 +1,88 @@
-From ae8a49ecc266430869f5ceff1200d5f3580d44e2 Mon Sep 17 00:00:00 2001
-From: Bastien Nocera <hadess@hadess.net>
-Date: Wed, 31 Jan 2024 16:07:34 +0100
-Subject: [PATCH 1/4] data: Fix long summary after Flathub review
+From 9476009d62145437f4fe3531fa17c7ffebc3d8da Mon Sep 17 00:00:00 2001
+From: Sabri Ünal <yakushabb@gmail.com>
+Date: Thu, 25 Jul 2024 23:32:25 +0300
+Subject: [PATCH] Fix appdata papercuts
```

Comment:
If those are all already upstream, could you please use commit `77954dc2b536170afb84d67c03d92dafefd3fc0d` as the basis to create a patch set, as was already done for those few fixes I submitted?